packages/dds/tree/src/index.ts

```diff
@@ -148,7 +148,7 @@ export {
 	SequenceFieldEditBuilder,
 	prefixPath,
 	prefixFieldPath,
-	cursorForJsonableTreeNode as singleTextCursor,
+	cursorForJsonableTreeNode,
```

Comment:
This seems unrelated to this PR

Reply:
I purged the previously accidental `as` renames in the top level exports, this was one of them. I basically removed or baked in all the renames, so this is kinda unrelated, but kinda related.